provider/terraform.rb

```diff
@@ -94,7 +94,7 @@ def updatable?(resource, properties)
 
     def force_new?(property, resource)
       !property.output &&
-        (property.input || (resource.input && property.update_url.nil? &&
+        (property.input || (resource.input && property.update_url.nil? && property.input.nil? &&
```

Comment:
I added this in here so that if a resource is `input: true`, a child property can overwrite that.  In this case, I added an Update method in the `resource_definition` that would allow `enabled` to be updated (even though `SecretVersion` has no update method, only `:enable` or `:disable`), when the resource technically can't be updated.  This is an assumption on my part that an explicit `input: false` on a child property can override this, but I'm not familiar with Ruby, or with all resources in the project to know if this will be problematic or not.

Reply:
Is there a reason to not mark the individual fields that cannot be updated as `input: true` and leave the `enabled` as the only one that is not `input` or `output`?

I think this would accomplish the same thing without weird input negation via overridden properties

Reply:
I tried that, however then it seems to generate the Update function, and in the generated Update function it makes a `Put` request, which isn't an option on the object.  So I needed an Update function that only called the expand function (because its in that function that I call the `:enable` or `:disable`.  Does that make sense?

Reply:
Ah yeah, and you can only have a single endpoint as the `update_url` on a field. In general you can have custom `update_url` and `update_verb` or something, but I don't think that will work in this case because it has to be `:enable` or `:disable` depending on the state